### DISCUSSION TITLE
how can i download picture

### DISCUSSION THREAD

**Discussion by 2504581605** [FlareSolverr/FlareSolverr#726] [Q&A]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: new version
- Last working FlareSolverr version:
- Operating system:
- Are you using Docker: [yes/no]
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no]
- Are you using a Proxy: [yes/no]
- Are you using Captcha Solver: [yes/no]
- If using captcha solver, which one:
- URL to test this issue:
```

### Description

i can use FlareSolverr get pagesource and post request,but how can i download picture
link : https://cdn1.sinobiological.com/image/label/tag_2023.png
thank you!

### Logged Error Messages

```text
Unable to get the byte stream of the image
```

**Comment by ngosang** [comment]
It's not supported yet.
https://github.com/FlareSolverr/FlareSolverr/issues/207
